Logix/Math/Constants/EpsilonFloat.cs

```diff
@@ -0,0 +1,11 @@
+﻿using BaseX;
+
+namespace FrooxEngine.LogiX.Math
+{
+    [Category("LogiX/Math/Constants")]
+    [NodeName("Epsilon")]
```

Comment:
maybe they should be named different things because this way it could get confusing

Reply:
Sure, I can make this change

Reply:
Done and done!